pkg/api/pod/util_test.go

```diff
@@ -3047,6 +3047,207 @@ func TestMarkPodProposedForResize(t *testing.T) {
 	}
 }
 
+func TestMarkPodProposedForResizeWithUnexpectedContainers(t *testing.T) {
```

Comment:
The testcase function for this looks identical to the previous test. Unless I'm missing something, just add these test cases to `TestMarkPodProposedForResize`

Reply:
Indeed, thank you.